src/content_script.ts

```diff
@@ -1,34 +1,33 @@
-import type { Message } from "../types";
+import type { Message } from "../devtools";
 
-// On message from page
+// On message from embed_script
 window.addEventListener("message", function (msg: MessageEvent<Message>) {
   const data = msg.data;
 
   // Forward onto devtools
   if (data.type === "GB_REFRESH") {
     chrome.runtime.sendMessage(data);
-  }
-  else if(data.type === "GB_ERROR") {
+  } else if (data.type === "GB_ERROR") {
     chrome.runtime.sendMessage(data);
   }
 });
 
 // On message from devtools
 chrome.runtime.onMessage.addListener(async (msg: Message) => {
-  // Forward onto page
+  // Forward onto embed_script
   if (msg.type === "GB_REQUEST_REFRESH") {
     window.postMessage(msg, "*");
   } else if (msg.type === "GB_SET_OVERRIDES") {
     window.postMessage(msg, "*");
   }
 });
 
-// Inject page script
-const SCRIPT_ID = 'gbdevtools-page-script';
-if(!document.getElementById(SCRIPT_ID)) {
-  const script = document.createElement('script');
+// Inject embed_script script
+const SCRIPT_ID = "gbdevtools-page-script";
+if (!document.getElementById(SCRIPT_ID)) {
+  const script = document.createElement("script");
   script.id = SCRIPT_ID;
   script.async = true;
-  script.src = chrome.runtime.getURL('static/js/page.js');
+  script.src = chrome.runtime.getURL("js/devtools_embed_script.js");
   document.body.appendChild(script);
-}
\ No newline at end of file
+}
```

Comment:
This may not be relevant for this set of changes, but I'm wondering if this approach will cause a conflict with more strict content security policies. Wondering if the Chrome [scripting](https://developer.chrome.com/docs/extensions/reference/scripting/) API's are more appropriate.

Reply:
At first glance it seems the benefit of chrome's scripting API is to be able to inject JS at 'extension runtime', whereas what we are doing above via content script happens 'on page load'. Both devtools and visual editor need the page to be rendered in order to work against the loaded site so I don't think we would want to use the scripting API in this case, correct me if I'm wrong.

In any case, you're right in that it's outside the scope of this PR ... Above is the existing implementation with a few tweaks of the original devtools repo; wouldn't want to change it just yet.

Reply:
That makes sense. I wasn't thinking about the time it is loaded but rather the content security policy. Either way, out of scope, and we will find out about a CSP error if/when someone reports one.